src/lib/support/SafePointerCast.h

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
```

Comment:
Fix date to 2021?

Reply:
Ah, yes. Good that CHIP is going to remove the dates. It's super annoying :)